src/aeser_api_encoder.erl

```diff
@@ -195,6 +196,7 @@ type2enc(signature)            -> ?BASE58;
 type2enc(commitment)           -> ?BASE58;
 type2enc(peer_pubkey)          -> ?BASE58;
 type2enc(name)                 -> ?BASE58;
+type2enc(subname)              -> ?BASE58;
```

Comment:
I understand that you take base58 because we did so for name... but base58 is more expensive in encoding. That means that we have an attack vector where it gets really slow for long names... is there a maximum length allowed that we check before encoding?

If you would have a name of 10000 bytes it would take a 500ms to encode. When having a name of 20000 bytes, it takes 2 seconds.

Reply:
I've changed it to BASE64.